Speech to Text/AudioFileViewController.swift

```diff
@@ -33,8 +33,7 @@ class AudioFileViewController: UIViewController, AVAudioPlayerDelegate {
         speechSample = Bundle.main.url(forResource: "SpeechSample", withExtension: "wav")
         player = try! AVAudioPlayer(contentsOf: speechSample)
         speechToText = SpeechToText(
-            username: Credentials.SpeechToTextUsername,
-            password: Credentials.SpeechToTextPassword
+            apiKey: Credentials.SpeechToTextApiKey
```

Comment:
I have a minor code style suggestion. Since the initializers are only using one parameter now, it would probably look better to make it one line: 
`speechToText = SpeechToText(apiKey: Credentials.SpeechToTextApiKey)`

Reply:
this one will go away because will keep the username and password as per your suggestion above